scripts/bash_aliases.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# This script will be sourced by the MIL runcom file if the install script
+# cloned this repository during a run or if the install script was run after
+# it was cloned manually. It contains general MIL commands that have been
+# aliased to make them faster or easier to execute. Becoming familiar with
+# these will likely increase productivity, so it is recommended to do so.
+
+# Source external alias modules
+source $CATKIN_DIR/src/software-common/scripts/two_line_bash.sh
+source $CATKIN_DIR/src/software-common/scripts/ros_connect.sh
+
+# Directory navigation
+alias swc="cd $CATKIN_DIR/src/software-common"
+
+# Bash sourcing
+alias srcbrc='source ~/.bashrc'
+
+# Catkin workspace management
+alias cm='catkin_make -C $CATKIN_DIR'
```

Comment:
I was under the impression that catkin_make automatically detected the number of available threads. Is this not the case?